src/helpers/utils.tsx

```diff
@@ -409,3 +420,122 @@ export function roundToPrecision(n: number, precision: number = 0): number {
   const factor = Math.pow(10, precision);
   return Math.round(n * factor) / factor;
 }
+
+/** Return columns with jsx implementations */
```

Comment:
Perhaps nitpicky, but what does "with jsx implementations" mean?  Is it possible to have a better docstring and function name here?